{"uid":"4e1a34ca2e02a542","name":"Валидация: создание борды с именем — ожидаемый статус 400","fullName":"tests.test_backend.board.board_validation_test#test_create_board_name_validation","historyId":"0ed5e545d8c0f89bd81250283d88d1f1","time":{"start":1749743622795,"stop":1749743623159,"duration":364},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":true,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"owner_client","time":{"start":1749743617438,"stop":1749743617914,"duration":476},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"base_url","time":{"start":1749743617438,"stop":1749743617438,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"_verify_url","time":{"start":1749743617438,"stop":1749743617438,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"delete_output_dir","time":{"start":1749743617438,"stop":1749743617438,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"pytestconfig","time":{"start":1749743617438,"stop":1749743617438,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_space","time":{"start":1749743617914,"stop":1749743618824,"duration":910},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_project","time":{"start":1749743618824,"stop":1749743620418,"duration":1594},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"passed","steps":[{"name":"Проверка, что API вернул статус 400 при number instead of string","time":{"start":1749743623159,"stop":1749743623159,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"attachments":[{"uid":"8946ebfcf985534c","name":"stdout","source":"8946ebfcf985534c.txt","type":"text/plain","size":91}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":1,"hasContent":true},"afterStages":[{"name":"temp_space::0","time":{"start":1749743651661,"stop":1749743652253,"duration":592},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"backend"},{"name":"parentSuite","value":"tests.test_backend.board"},{"name":"suite","value":"board_validation_test"},{"name":"host","value":"pkrvmxyh4eaekms"},{"name":"thread","value":"2521-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_backend.board.board_validation_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"expected_status","value":"400"},{"name":"name","value":"123"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":6,"skipped":0,"passed":1,"unknown":0,"total":7},"items":[{"uid":"8c497ae3a02cb0c9","reportUrl":"https://vaizcom.github.io/autotests/119//#testresult/8c497ae3a02cb0c9","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fc7b5913490>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742819883,"stop":1749742819883,"duration":0}},{"uid":"c02d0e2233dad1f0","reportUrl":"https://vaizcom.github.io/autotests/118//#testresult/c02d0e2233dad1f0","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9cb88f1f10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742667257,"stop":1749742667257,"duration":0}},{"uid":"6b102dfa1d2506a8","reportUrl":"https://vaizcom.github.io/autotests/117//#testresult/6b102dfa1d2506a8","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f2b9f21f250>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742497013,"stop":1749742497013,"duration":0}},{"uid":"29ee04e0e3f2125c","reportUrl":"https://vaizcom.github.io/autotests/116//#testresult/29ee04e0e3f2125c","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7efcc7f78ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749741233476,"stop":1749741233476,"duration":0}},{"uid":"b6c7dc4f6e84a0e1","reportUrl":"https://vaizcom.github.io/autotests/115//#testresult/b6c7dc4f6e84a0e1","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f28d1ddcd10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740825946,"stop":1749740825946,"duration":0}},{"uid":"9450d640bb230b0","reportUrl":"https://vaizcom.github.io/autotests/114//#testresult/9450d640bb230b0","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f271b110ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740652720,"stop":1749740652720,"duration":0}}]},"tags":["backend"]},"source":"4e1a34ca2e02a542.json","parameterValues":["400","123"]}